homeassistant/components/nightscout/sensor.py

```diff
@@ -0,0 +1,139 @@
+"""Support for Nightscout sensors."""
+from asyncio import TimeoutError as AsyncIOTimeoutError
+from datetime import timedelta
+import hashlib
+import logging
+from typing import Callable, List
+
+from aiohttp import ClientError
+from py_nightscout import Api as NightscoutAPI
+import voluptuous as vol
+
+from homeassistant.components.sensor import PLATFORM_SCHEMA
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.const import CONF_NAME, CONF_RESOURCE
+from homeassistant.core import HomeAssistant
+import homeassistant.helpers.config_validation as cv
+from homeassistant.helpers.entity import Entity
+
+from .const import ATTR_DATE, ATTR_DELTA, ATTR_DEVICE, ATTR_DIRECTION, ATTR_SVG, DOMAIN
+
+SCAN_INTERVAL = timedelta(minutes=1)
+
+_LOGGER = logging.getLogger(__name__)
+
+DEFAULT_NAME = "Blood Glucose"
+
+PLATFORM_SCHEMA = PLATFORM_SCHEMA.extend(
```

Comment:
Since this is a new integration, doesnt need to support the old platform style schema